app/Services/UserService.php

```diff
@@ -32,7 +35,25 @@ public function updateUserEmail($email, $user)
     public function getUsers()
     {
         // Get emails from users table
-        return  User::all()->pluck('email')->toArray();;
+        return User::all()->pluck('email')->toArray();;
     }
 
+    public function delete($user)
+    {
+        try {
+        // change user status
+        $user->status = User::INACTIVE;
+        // genrate pdf
+        $user->save();
+        $pdf = new Dompdf();
+        $pdf->loadHTML("<h1>User successfully deleted</h1>");
+        $pdf->setPaper('A4', 'portrait');
+        $pdf->render();
+        // send mail
+        Mail::to($user->email)->send(new DeleteMailer($pdf));
+        } catch (\Exception $e) {
+            // handle exception
+            return response()->json(['status' => 'error', 'message' => 'Failed to delete user']);
```

Comment:
service method shouldn't return 'response'. Move the try-catch block to the controller action